angular-app/src/app/workspace/workspace.component.html

```diff
@@ -35,10 +35,9 @@ <h1 id="audio-name" class="center">Select an Audio File</h1>
             <div id="bottom-content">
                 <h1 id="lyric-doc-name" class="center">{{project.lyricsFileName}}</h1>
                 <br/>
-                <textarea id="lyric-doc" placeholder="Type Here..." rows="50" cols="100"></textarea>
+                <textarea id="lyric-doc" placeholder="Type Here..." rows="50" cols="100">{{project.lyricsContent}}</textarea>
                 <br/>
                 <button mat-flat-button color="primary" class="center" (click)="saveLyrics()">Save</button>
```

Comment:
This looks good, but as noted in @gwynethsc, the way the project saves may overwrite these changes. You all should think about just turning this Save button into a `saveProject()` instead of just `saveLyrics()`. But, given your time constraint, I would leave as is and add a Issue to fix later if possible.